Makefile

```diff
@@ -312,7 +312,7 @@ endif
 	flake8 test/deps/import-busybox
 	shellcheck --shell sh test/*.sh test/includes/*.sh test/suites/*.sh test/backends/*.sh test/lint/*.sh
 	shellcheck test/extras/*.sh
-	run-parts --verbose --exit-on-error --regex '.sh' test/lint
+	run-parts $(shell run-parts -V 2> /dev/null 1> /dev/null && echo -n "--verbose --exit-on-error --regex '.sh'") test/lint
```

Comment:
Why not the traditional `>/dev/null 2>&1`?

Reply:
done